helm/ngrok-operator/templates/rbac/bindingconfiguration_editor_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to edit bindingconfigurations
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    {{- include "ngrok-binding.labels" . | nindent 4 }}
+    app.kubernetes.io/component: rbac
+  name: {{ include "ngrok-binding.fullname" . }}-bindingconfiguration-editor-role
```

Comment:
```suggestion
  name: {{ include "ngrok-operator.fullname" . }}-bindingconfiguration-editor-role
```

Reply:
We'll probably want this change on the rest of them as well. Otherwise, looks great! :clap:

Reply:
Yep missed that but CI caught it. Accidentally renamed with my search/replace.